deploy/kustomize/deployment.yaml

```diff
@@ -15,7 +15,7 @@ spec:
       serviceAccountName: hiphops-worker
       containers:
       - name: hiphops
-        image: hiphops/hiphops:v0.6.1
+        image: hiphops/hiphops:v0.11.0
```

Comment:
`0.10.0` requires a config.yaml file to be present.

Reply:
We change these after a release is cut, not before